CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.12.0] - released on 2017-05-31
+### Added
+- Added `generate_source_artifacts` method to the platform dsl to allow generation
+  of `.src.rpm` and debian source artifacts.
+- Added the ability to specify mirror URLs.
```

Comment:
I'd probably change where the bulk of the detail lives. I cribbed from your Deprecated writeup and moved it into Added:

```markdown
- Added support to the Component DSL for Mirror URLs using the `#mirror` method. This functionality supersedes the previous Rewrite Rule functionality and the Rewrite Rule functionality has been deprecated (see *Deprecated* for `0.12.0`).

  Vanagon will attempt to use any Mirror URLs provided for a component before it attempts to use the canonical upstream URL. Multiple Mirror URLs can be specified, and Vanagon will attempt to retrieve them in random order until a URL returns a `200` code or all Mirror URLs are exhausted.
```